controllers/profiles/getOtherProfileByUsername.js

```diff
@@ -54,7 +54,7 @@ module.exports = async (req, res) => {
     });
 
     const isFollowing = await sequelize.query(isFollowingQuery, {
-      replacements: {user_id_follower: req?.userId || '', user_id_followed: targetUserId}
+      replacements: {user_id_followed: req?.userId || '', user_id_follower: targetUserId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the order of `user_id_followed` and `user_id_follower` might affect the logic of the query. Please ensure that the SQL query is expecting these parameters in this new order. If not, it could lead to incorrect results or even a potential security issue if user permissions are involved.
```diff
-      replacements: {user_id_follower: req?.userId || '', user_id_followed: targetUserId}
+      replacements: {user_id_followed: req?.userId || '', user_id_follower: targetUserId}
```
If the SQL query expects the parameters in the original order, you should revert this change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->